script.sh

```diff
@@ -99,7 +99,29 @@ else
     gerror
 fi
 
-cp -rH files/"${style}"/Numix/* "${dir}"/Numix/
+read -p "Which folder colour do you want? " answer
+if [[ "default,blue,green,grey,orange,pink,purple,red,yellow" == *$answer* ]]; then
```

Comment:
I have a plan for this similar to how we do it for style, so don't worry about that :)